Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hades252"
-version = "0.10.1"
+version = "0.10.2"
```

Comment:
The changes in BLS and plonk are breaking. I believe this one should be a breaking change too